webdriver/tests/bidi/network/response_completed/response_completed.py

```diff
@@ -190,6 +191,7 @@ async def test_response_headers(wait_for_event, url, fetch, setup_network_test):
     assert_response_event(
         events[0],
         expected_request=expected_request,
+        expected_response=expected_response,
```

Comment:
Not sure whether this should be added or if the variable assignment should be removed.

Reply:
ditto